dozer-ingestion/src/connectors/postgres/helper.rs

```diff
@@ -153,6 +157,11 @@ pub fn value_to_field(
             })
         }
         &Type::POINT => convert_row_value_to_field!(row, idx, GeoPoint),
+        // &Type::UUID => convert_row_value_to_field!(row, idx, Uuid),
+        &Type::UUID => {
```

Comment:
@karolisg is it possible to refactor the code so we don't duplicate the conversion code? Now we have `value_to_field` and `postgres_type_to_field` doing the same thing.

Reply:
currently, 

```rust
pub fn value_to_field(
    row: &Row,
    idx: usize,
    col_type: &Type,
) -> Result<Field, PostgresSchemaError> {
```

```rust 
pub fn postgres_type_to_field(
    value: Option<&Bytes>,
    column: &TableColumn,
) -> Result<Field, PostgresSchemaError> {
```

I was thinking replacing `postgres_type_to_field` with `value_to_field` but we don't have access to `Row` where we are calling `postgres_type_to_field()` at the moment, this might be a bigger refactor than we think.

Reply:
> this might be a bigger refactor than we think

True. And neither of us is familiar enough with that part of code, so I was asking @karolisg. It won't be part of this PR.